wkconnect/backends/boss/token_repository.py

```diff
@@ -69,3 +82,21 @@ def __init__(self, http_client: ClientSession):
 
     async def get_header(self, dataset: DatasetDescriptor) -> Dict[str, str]:
         return {"Authorization": await self.get(dataset)}
+
+    async def logout(self, dataset: DatasetDescriptor) -> None:
+        key = self.__make_key(dataset)
+        url = self._openid_url(key) + "logout"
+        data = {
+            "refresh_token": self.token_infos[key]["refresh_token"],
+            "client_id": self.client_id,
+        }
+        try:
+            await self.http_client.post(url, data=data)
+            del self.token_infos[key]
+        except ClientResponseError:
+            pass
```

Comment:
Is this ClientResponseError harmless enough to just pass? Maybe add some logging?

Reply:
I would say so. The most common case this will happen will be that the token is expired, which I didn't want to check upfront, but rather simply except here. I'll add a comment to explain this.
Is this fine iyo? I can also add logging, but didn't want to clutter the output with stuff that won't harm too much anyway.

Reply:
sure, it’s fine =)